apps/web/config/chains.ts

```diff
@@ -27,7 +27,7 @@ export const CHAIN_PARAMS: Record<Network, ChainParams> = {
     chainName: 'Dango Testnet',
     nativeCurrency: { name: 'Dango Celo', symbol: 'D-CELO', decimals: 18 },
     rpcUrls: ['https://forno.dango.celo-testnet.org'],
-    blockExplorerUrls: [],
+    blockExplorerUrls: ['https://celo-dango.blockscout.com/'],
```

Comment:
Should we remove slash at the end since  blockExplorerUrls: ['https://explorer.celo.org/alfajores'],

Reply:
Good catch, removed